test/ops/test_install.rb

```diff
@@ -25,6 +25,23 @@ module Ops
                     gemfile_source: gemfile_source
             end
 
+            it "may install non-interactively" do
```

Comment:
ruby test/ops/test_install.rb fails 
```
Errno::ENOENT: No such file or directory @ dir_s_mkdir  .../autoproj/lib/autoproj/../../vendor/test_gem_home
    /usr/lib/ruby/2.5.0/fileutils.rb:1265:in `mkdir'
   ...
```
anything i am missing?

Reply:
It seems that the tests assume you've checked out your autoproj installation separately, and bundle-installed it with `--path=vendor`.

Could you try it this way ? If that solves the issue, I'll add some documentation in README for it.

Reply:
Sorry, not clear how I should properly test now.
Wouldn't ensuring the existence of the directory by fixing Test::fixture_gem_home in lib/autoproj/test.rb be sufficient? 
If doing that the test starts running,  but still gives error: 

```
  1) Error:
Autoproj::Ops::Install::default shared gems location#test_0004_sets the environment to point RubyGems to the shared location:
Gem::UnsatisfiableDependencyError: Unable to resolve dependency: user requested 'did_you_mean (= 1.2.0)'
```

Reply:
The code assumes that the gem location is in `vendor/`

Which happens if you install autoproj (for testing) with bundler, that is:

~~~
git clone https://github.com/rock-core/autoproj
cd autoproj
bundle install --path=vendor
bundle exec ruby test/ops/test_install.rb
~~~